doc/dev_guide/contributing.rst

```diff
@@ -229,3 +229,35 @@ We use `GitHub Actions <https://github.com/pyxem/pyxem/actions>`_ to ensure that
 can be installed on Windows, macOS and Linux (Ubuntu). After a successful installation,
 the CI server runs the tests. After the tests return no errors, code coverage is
 reported to `Coveralls <https://coveralls.io/github/pyxem/pyxem?branch=main>`_.
+
+Making a release
+================
+
+We use `GitHub Actions <https://github.com/pyxem/pyxem/actions>`_ to automatically
+create a new release. Each time a new tag is pushed to the repository, the CI server
+will:
+
+1. Build the documentation and upload it to `Read the Docs
```

Comment:
I believe new tags, signifying new releases, will be uploaded to "stable". "latest" is reserved for builds from "main", right?

Reply:
Based on the read the docs versioning you are correct.  I should probably clarify this a little bit.  For reference: https://docs.readthedocs.io/en/stable/versions.html#versioned-documentation